mlos_bench/mlos_bench/tests/storage/trial_data_test.py

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests for loading the trial metadata.
+"""
+
+# TODO
```

Comment:
This is a stub for now.  There were no tests for TrialData loading previously.  I didn't really change that code yet, so didn't add tests for it yet, but wanted to note that we need some while I was adding some for the ExperimentData class.